Formula/qt.rb

```diff
@@ -111,17 +132,39 @@ class Qt < Formula
     directory "qtbase"
   end
 
-  # Patch for https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-25255
+  # Apply upstream commit to fix build on older Linux systems
   patch do
-    url "https://download.qt.io/official_releases/qt/6.2/CVE-2022-25255-qprocess6-2.diff"
-    sha256 "62068c168dd1fde7fa7b0b574fcb692ca0c20e44165fa2aafae586ca199b9f00"
+    url "https://invent.kde.org/qt/qt/qtbase/-/commit/311d29d2261a7e0689340c4c1322138f8234da7b.diff"
+    sha256 "a9cda53f71d307be9b35cbd60a56d369e5611dcf0bee3246c9f2f34b128928de"
     directory "qtbase"
   end
 
+  # Apply upstream commit to fix using system icu in chromium.
+  patch do
+    url "https://code.qt.io/cgit/qt/qtwebengine-chromium.git/patch/?id=75f0f4eb"
+    sha256 "ec28b71135f293f624365a50be0c329e396eaa9433655386af146614837e82a2"
+    directory "qtwebengine/src/3rdparty"
+  end
+
+  # Apply patch to fix chromium build with glibc < 2.27. See here for details:
+  # https://libc-alpha.sourceware.narkive.com/XOENQFwL/add-fcntl-sealing-interfaces-from-linux-3-17-to-bits-fcntl-linux-h
+
+  patch :DATA
+
   def install
+    # Install python dependencies for QtWebEngine
+    venv = virtualenv_create(libexec, "python3")
```

Comment:
[Document](https://doc.qt.io/qt-6/qtwebengine-platform-notes.html#all-platforms) said `html5lib` is build time dependency.

Reply:
Ah good catch there, this should go into a `venv` in `buildpath` instead.  This is pretty easy to test quickly as it checks if these dependencies are found very early in the build process.

Reply:
I moved the `virtualenv` into `buildpath` and the dependencies were found successfully, so this is now fixed.